app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
+
+    def add_husband_wife_if_merried(self, person: dict) -> None:
+        if person.get("wife") is not None:
```

Comment:
It is not necessary to write "is not None" in this case.

Reply:
noted, thanks